tsconfig-build.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "./tsconfig.json",
-  "exclude": ["src/**/*.stories.*", "src/**/*.spec.*"]
+  "compilerOptions": {
+    "rootDir": "./src/",
```

Comment:
comment (non-blocking): I don't think we need the trailing `/` here but it's not critical so happy for you to leave

Reply:
I never really know if it's needed but it's normally to specify the trailing `/` to make clear that something's a directory, so imo it does no harm and is potentially beneficial - unless you know a reason it's better not to have it?

tbh I think I put it there mainly beacuse the `outDir` already has a trailing slash in our tsconfig and I assumed they would use the same convention 🤷